index.html

```diff
@@ -18,11 +18,43 @@
 <body>
   <header>
     <div class="container">
+      <h3>Code Journal</h3>
```

Comment:
There should be a `row` and a `column-` of some sort holding this. Please use layout classes properly.